docs/products/redis/concepts/ssl-connectivity.rst

```diff
@@ -0,0 +1,85 @@
+SSL connectivity
+================
+
+Client support for SSL-encrypted connections
+--------------------------------------------
+
+Default support
+~~~~~~~~~~~~~~~
+Aiven for Redis uses SSL encrypted connections by default. This is shown by the use of ``rediss://`` (with double s) prefix in the ``Service URI``. 
+
+.. Tip::
+    You can find the ``Service URI`` on `Aiven console <https://console.aiven.io/>`_.
+
+Since **Redis 6**, the redis-cli tool itself supports SSL connections; therefore, you can connect directly to your service using::
```

Comment:
```suggestion
Since **Redis 6**, the ``redis-cli`` tool itself supports SSL connections; therefore, you can connect directly to your service using::
```